en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -15,12 +15,13 @@ We'll go through each of these in detail in the following sections.
 | Main interface | Function |
 | :--- | :--- |
 | Power | Connect Power module;  power input & AD voltage  and current detection. |
-| PM2 | Connect i2c smart battery. |
-| TF CARD | Insert SD card for log storage |
+| PM2 | [Do not use with PX4](../flight_controller/cuav_v5_nano.md#issue_pm2) |
+| TF CARD | SD card for log storage (comes with card) |
 | M1~M8 | PWM outputs. Can be used to control motors or servos. |
-| A1~A8 | PWM outputs. Can be used to control motors or servos. |
+| A1~A3 | PWM outputs. Can be used to control motors or servos. |
```

Comment:
@davids5 Is this accurate? The original version was copied from v5+ ; all I have done is removed the A4 - 8 port info because I can't see those ports on the board.

Reply:
The Nano does not have a PX4IO so Main is a misnomer, but in this context I guess it is OK.
A1-A3 are Caputre pins. There is no Capture driver in PX4 for them yet. Nor is there a plan to use them as PWM driver so they Can NOT be used to control motors or servos. (unless some added that feature but it was not our usecase) See https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165&range=A8:G8

Reply:
Thanks @davids5 . I'll updat the line to say (OK?):
```
| A1~A3 | Capture pins (not supported on PX4). |
```
Assumption is that if it says this no one will assume they can be used. 

> The Nano does not have a PX4IO so Main is a misnomer, but in this context I guess it is OK.

I tend to use MAIN and AUX because the airframe reference uses those, and so generally does the screenprinting. If I refer to something else then we need to update the airframe reference to remove the disconnect.

In any case how do you refer to these?